src/common/schedulers/slurm_commands.py

```diff
@@ -110,16 +110,21 @@ def update_nodes(
             node_info += f" nodeaddr={addrs}"
         if hostnames:
             node_info += f" nodehostname={hostnames}"
-        run_command(  # nosec
```

Comment:
Have we validated the arguments used when building the command? E.g. `state`, `reason` etc?